### PR TITLE
Print a proper error if the version not found in the .ocamlformat file

### LINKED ISSUE #388: If the version is not listed in `.ocamlformat`, lint-fmt will fail.
If the version is not listed in `.ocamlformat`, lint-fmt will fail.
This is due to the fact that if version is not specified, version will be `undefined`.

https://github.com/HikaruEgashira/sat-solver/runs/4797615555?check_suite_focus=true#step:7:14

### PR DESCRIPTION
**Pull request by HikaruEgashira**
print "version not found in .ocamlformat" if the version doesn't exist.

resolve #388

### CODE REVIEW

File: src/lint-fmt/ocamlformat.ts [hunk 1]
```diff
@@ -18,5 +18,8 @@ async function parse() {
 export async function getOcamlformatVersion() {
   const config = await parse();
   const version = config.filter((line) => line[0] === "version").flat()[1];
+  if (version === undefined) {
+    throw new Error("version not found in .ocamlformat");
```
Comment (smorimoto) [inline src/lint-fmt/ocamlformat.ts:22]:
```suggestion
    throw new Error("Field version not found in .ocamlformat file: setting up your project to use the default profile and the OCamlFormat version you installed in .ocamlformat file is considered good practice");
```

**Comment by smorimoto** [comment]
Looks good! I left a small suggestion, please check it. In addition, could you add the changelog entry?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [unreleased]
 
+### Fixed
+
+- Outputs an error if the version is not specified in the `.ocamlformat` file.
```
Comment (smorimoto) [inline CHANGELOG.md:13]:
```suggestion
- Print a proper error if the version not found in the `.ocamlformat` file.
```

**Review by smorimoto — APPROVED** [review]

File: lint-fmt/index.js [hunk 3]
```diff
@@ -7375,6 +7375,9 @@ function getOcamlformatVersion() {
                 case 1:
                     config = _a.sent();
                     version = config.filter(function (line) { return line[0] === "version"; }).flat()[1];
+                    if (version === undefined) {
+                        throw new Error("version not found in .ocamlformat");
```
Comment (smorimoto) [inline lint-fmt/index.js:7379]: The build output does not look updated properly.

**Comment by smorimoto** [comment]
Thank you @HikaruEgashira!

**Comment by HikaruEgashira** [comment]
Thank you for review!!

**Pull request merged**